server/middlewares/rateLimiter.ts

```diff
@@ -35,7 +41,20 @@ export function rateLimiter() {
   };
 }
 
-export function registerRateLimiter(config: RateLimiterConfig) {
+type RateLimiterConfig = {
```

Comment:
Config applicable to individual routes is minimal and `points` is mapped to `requests`

Reply:
Thanks :+1:  I was about to suggest this because `points` didn't make any sense in this context.

Reply:
Thanks +1 I was about to suggest this because points didn't make any sense contextually.